app/run.go

```diff
@@ -22,6 +43,46 @@ func Run() error {
 	return s.Serve()
 }
 
+func setupLogger() (func(), error) {
+	var (
+		cfg  zap.Config
+		opts []zap.Option
+	)
+
+	switch os.Getenv("APP_ENV") {
+	case "production":
+		cfg = zap.NewProductionConfig()
+		cfg.Level = zap.NewAtomicLevelAt(zapcore.WarnLevel)
+		cfg.DisableStacktrace = true
+	default:
+		cfg = zap.NewDevelopmentConfig()
+		cfg.EncoderConfig.EncodeLevel = zapcore.CapitalColorLevelEncoder
+		cfg.Level = zap.NewAtomicLevelAt(zapcore.DebugLevel)
+		cfg.EncoderConfig.EncodeTime = func(t time.Time, enc zapcore.PrimitiveArrayEncoder) {
+			enc.AppendString(t.Local().Format("2006-01-02 15:04:05 MST"))
+		}
+		cfg.DisableStacktrace = true
+	}
+
+	l, err := cfg.Build(opts...)
+
+	if err != nil {
+		return nil, fail.Wrap(err)
+	}
+
+	var closers []func()
+	closers = append(closers, func() { l.Sync() })
```

Comment:
**[golangci]** <sub>reported by [reviewdog](https://github.com/haya14busa/reviewdog) :dog:</sub>
Error return value of `l.Sync` is not checked (errcheck)